frontend/src/metabase/visualizations/echarts/cartesian/model/index.ts

```diff
@@ -131,6 +137,7 @@ export const getCartesianChartModel = (
     dataset,
     transformedDataset,
     seriesModels,
+    yAxisScaleTransforms,
```

Comment:
For x-axis they are the part of the axis model which can be a bit more convenient due to less parameters needed to pass around and also consistent

Reply:
Yep, sorry, I meant to just put the transforms into the axis but use it separately before creating axis models since they depend on the transformed data. Would this work or you think it is a bad idea?

Reply:
It does work but I don't think there's too much benefit to it. If we're creating them separately before hand I think we can leave them outside of the axis model since they are already a top level object in `getCartesianChartModel`. For now I'll go with this approach, and maybe if there's a reason to put them in the axis model later I'll move them.